3_visualize/src/plot_gradient.R

```diff
@@ -51,6 +51,10 @@ plot_gradient <- function(ensemble_data, site_info, date_start, days_shown, out_
     scico::scale_color_scico(palette = "lapaz", 
                             end = 0.5,
                             na.value = "orangered") +
+    # change alpha so that end of confidence interval shows and doesn't fade away
+    scale_slab_alpha_continuous(
+      range = c(.15, 1) #default: 0,1
+    ) +
```

Comment:
Wow I was unsure about how much a difference this would have, but it looks pretty rad.  I am a bit confused on how this works though, since we haven't mapped `alpha` to the data anywhere. If that inherent to `stat_gradientinterval`?

Reply:
Yes, apparently it's inherent to the `stat_gradientinterval', from what I read. And the default is to go all the way to `alpha = 0`